### ISSUE TITLE
Nuget package contains Debug build of the library

### ISSUE THREAD

**Issue by ied206** [invertedtomato/crc#1]
Hello, I have been trying to benchmark several C# and native implementations of CRC32 and CRC64. However, I found BenchmarkDotNet complaining about `non-optmized` binaries of `InvertedTomato.Crc`. Please update the nuget packages to the Release build, so people can use the optimized build.

+ This library was the one of few libraries where I could find the CRC64 implementations. Thanks for your work!

```
C:\Joveler\Joveler.Compression\Benchmark>dotnet run -c Release -- hash
// Validating benchmarks:
Assembly Benchmark which defines benchmarks references non-optimized InvertedTomato.Crc
        If you own this dependency, please, build it in RELEASE.
        If you don't, you can disable this policy by using 'config.With(ConfigOptions.DisableOptimizationsValidator)'.
```

**Comment by invertedtomato** [comment]
Ah gee, good call! I've updated Nuget with the optimized binaries now. Thanks for pointing that out.

**Comment by ied206** [comment]
The issue was solved in 1.1.3, thanks!
